scripts/helpers.sh

```diff
@@ -85,5 +85,5 @@ notify() {
   # trigger visual bell
   # your terminal emulator can be setup to set URGENT bit on visual bell
   # for eg, Xresources -> URxvt.urgentOnBell: true
-  tmux split-window "echo -e \"\a\" && exit"
+  tmux split-window -t "\$$SESSION_ID":@"$WINDOW_ID" "echo -e \"\a\" && exit"
```

Comment:
```suggestion
  tmux split-window -t "${SESSION_ID}:@${WINDOW_ID}" "echo -e \"\a\" && exit"
```

Should also work.

Reply:
This does not work considering `$` is a tmux prefix for [delineating session id's](https://man7.org/linux/man-pages/man1/tmux.1.html#COMMANDS) like how `@` is a prefix for delineating window id's and `#` delineates pane id's. So the escaped `$` is required, otherwise the tmux command target assumes we're talking about a session name rather than an ID.

```shell
tmux split-window -t "\$${SESSION_ID}:@${WINDOW_ID}" "echo -e \"\a\" && exit"
```
Is an alternative if you prefer this way of formatting for the variables, but since the variable will always be an integer it's not strictly necessary in this situation.

Reply:
> This does not work considering `$` is a tmux prefix for [delineating session id's](https://man7.org/linux/man-pages/man1/tmux.1.html#COMMANDS) like how `@` is a prefix for delineating window id's and `#` delineates pane id's. So the escaped `$` is required, otherwise the tmux command target assumes we're talking about a session name rather than an ID.
> 
> ```shell
> tmux split-window -t "\$${SESSION_ID}:@${WINDOW_ID}" "echo -e \"\a\" && exit"
> ```
> 
> Is an alternative if you prefer this way of formatting for the variables, but since the variable will always be an integer it's not strictly necessary in this situation.

Ah, thanks for the explanation. Makes since I think I did not test all use-cases 👍🏻.